Documentation/ApiOverview/Logging/Logger/Index.rst

```diff
@@ -241,11 +243,13 @@ Registration via class attribute for :php:`LoggerInterface` injection:
      }
    }
 
-Registration via parameter attribute for :php:`LoggerInterface` injection,
-overwrites possible class attributes:
+Registration via parameter attribute for :php:`LoggerInterface`
```

Comment:
Hmm, we inject not a LoggerInterface but a logger. LoggerInterface is only a type hint, DI takes care about injecting the concrete logger. Also I don't understand "overwrites possible class attributes".

Maybe better: "We use constructor injection to get an instance of a logger:"

Reply:
@brotkrueml I think this refers to a part in the docs which was already there before. The part

> Registration via parameter attribute for LoggerInterface injection, overwrites possible class attributes:

is already on the current documentation page, see https://docs.typo3.org/m/typo3/reference-coreapi/11.5/en-us/ApiOverview/Logging/Logger/Index.html

So, I don't really want to change too much which was not part of this PR, which was only for adding how to use the channel in the configuration. I did change some more stuff where it otherwise would not have made sense or to unify it, but only where it directly affected my change.

About the LoggerInterface - this is the type which is used, and not Logger, so I think it is more precise to use this than just to use "logger" which is also ok, but does not give us any information which type of interface / class is used. I think it is correct to say we inject LoggerInterface or we inject an object of type LoggerInterface or an object of a class which implements LoggerInterface, but the third option may be more precise, but it starts to get awkward.

Anyhow, unless I don't introduce any new error and there are some things where we would have to rework the entire section, I would rather get this merged now.

Reply:
I disagree, an interface cannot be injected, only an object - also the wording must reflect this. But as this chapter must be reworked anyway, I am fine to postpone that.